packages/edit-post/src/components/visual-editor/style.scss

```diff
@@ -1,18 +1,16 @@
 .edit-post-visual-editor {
 	position: relative;
-	padding: 50px 0;
+	padding-top: 50px;
 
 	& .components-button {
 		font-family: $default-font;
 	}
 }
 
 .edit-post-visual-editor .block-editor-writing-flow__click-redirect {
-	// Collapse to minimum height of 50px, to fully occupy editor bottom pad.
-	height: 50px;
+	// Allow the page to be scrolled with the last block in the middle.
```

Comment:
I suppose it's intended then that there's a large gap at the bottom of a scrollable post?

![image](https://user-images.githubusercontent.com/1779930/62221893-6e58a600-b380-11e9-9912-bf7f80172107.png)

Reply:
Yes, that's by intention